assets/queries/dockerfile/disable_pip_cache/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "disable_pip_cache",
+  "queryName": "Disable Pip Cache",
```

Comment:
The query name should reflect the vulnerability, not the fix

Reply:
Fixed in https://github.com/Checkmarx/kics/pull/1410